sqlitedict.py

```diff
@@ -140,6 +141,8 @@ def __init__(self, filename=None, tablename='unnamed', flag='c',
         object.
         The default is to use pickle.
 
+        If you disable `outer_stack`, the stacktrace at insert time won't be saved. This
```

Comment:
More context / info please. As a user, what is "stacktrace at insert time", when would I need that and why?